docs/08-贡献指南/29how-to-contribute.md

```diff
@@ -13,19 +13,21 @@ Apache RocketMQ —— 开放共享的开源社区，诚挚邀请您的加入
 - 测试预发布版本
 
 
-## 提出问题！
+## 请求答疑
 
-Apache RocketMQ 社区提供完备的处理流程帮您解答问题。 您可以通过 [user mailing list](https://rocketmq.apache.org/about/contact/) 和 [apache-rocketmq](https://stackoverflow.com/questions/tagged/rocketmq) 进行提问。
+Apache RocketMQ 社区提供完备的处理流程帮您解答问题。
+
+您可以通过 [user mailing lists](mailto:users@rocketmq.apache.org) 和 [stackoverflow rocketmq](https://stackoverflow.com/questions/tagged/rocketmq) 进行提问。
```

Comment:
user mailing lists -> user mailing list
stackoverflow rocketmq -> Stack Overflow #rocketmq

Reply:
Thx, it seems more better